.eslintignore

```diff
@@ -2,4 +2,5 @@
 **/locales/index.js
 cli/config/init.entrypoint.js
 cli/config/init.App.test.js
-
+shell/src/D2App/*
+examples
```

Comment:
Why are we excluding `examples` from being linted?

Reply:
I had previously encountered linting errors regarding unresolved imports due to './locales' not existing relative to `examples/simple-app/src/App.js` and did not want to include a `// eslint-disable-line import/unresolved` comment in an example. https://github.com/dhis2/app-platform/blob/9aea659cde05400b96bc0afca18c55ca6c0e886e/examples/simple-app/src/App.js#L2
This issue doesn't seem to appear anymore though when I remove `examples` from `.eslintignore` :man_shrugging: